tiledb/sm/array_schema/dimension_label.cc

```diff
@@ -139,6 +139,7 @@ DimensionLabel::DimensionLabel(
           label_type == Datatype::STRING_ASCII ? constants::var_num : 1)
     , schema_(make_shared<ArraySchema>(
           HERE(),
+          make_shared<MemoryTracker>(HERE()),
```

Comment:
This won't compiler after a rebase, so you'll have to add the MemoryTracker argument and follow that up the stack.